Tests/PackageLoadingTests/ToolsVersionLoaderTests.swift

```diff
@@ -38,83 +33,83 @@ class ToolsVersionLoaderTests: XCTestCase {
-            "\u{B}  \u{200A}\u{D}\u{A}\t\u{85}\u{85}\u{A}\u{2028}\u{2009}\u{2001}\u{C}//sWiFt-tOoLs-vErSiOn:9995.5.2\nkkk\n"                 : (9995, 5, 2, "9995.5.2"),
-            // Some leading whitespace characters, and an assortment of horizontal whitespace characters surrounding the label for Swift ≥ Next:
-            "\u{2002}\u{202F}\u{A}//\u{A0}\u{1680}\t\u{2004}\u{2001} \u{2002}swift-tools-version:\u{3000}9995.4.0"       : (9995, 4, 0, "9995.4.0"),
-            "\u{B}//\u{A0}\u{1680}\t\u{2000}\u{2001} \u{2002}swift-tools-version:\u{202F}\u{3000}9995.4-dev"             : (9995, 4, 0, "9995.4.0"),
-            "\u{C}//\u{A0}\u{1680}\t\u{2000}\u{2001} swift-tools-version:\u{2002}\u{202F}\u{3000}9995.8.0"               : (9995, 8, 0, "9995.8.0"),
-            "\u{D}//\u{A0}\u{1680}\t\u{2005} \u{202F}\u{3000}swift-tools-version:\u{2001}9995.8.0-dev.al+sha.x"          : (9995, 8, 0, "9995.8.0"),
-            "\u{D}\u{A}//\u{A0}\u{2001} \u{2002}\u{202F}\u{3000}swift-tools-version:\u{1680}\t\u{2000}9996.1.2"          : (9996, 1, 2, "9996.1.2"),
-            "\u{85}//\u{2000}\u{2001} \u{2006}\u{202F}\u{3000}swift-tools-version:\u{A0}\u{1680}\t9996.1.2;"             : (9996, 1, 2, "9996.1.2"),
-            "\u{2028}//\u{2001} \u{2002}\u{2007}\u{3000}swift-tools-vErsion:\u{A0}\u{1680}\t\u{2000}9996.1.2;;;;;"       : (9996, 1, 2, "9996.1.2"),
-            "\u{2029}//\u{202F}\u{3000}swift-tools-version:\u{A0}\u{1680}\t\u{2000}\u{2001} \u{2002}9996.1.2;x;x;x;x;x;" : (9996, 1, 2, "9996.1.2"),
-            "\u{A}\u{D}\u{85}\u{202F}\u{2029}\u{A}\u{2028}//\u{2000}\u{2001}\u{9}swift-toolS-version:\u{A0}\u{1680}\t\u{2000}\u{2009} \u{2002}\u{202F}9995.5.2;hello" : (9995, 5, 2, "9995.5.2"),
-            "\u{D}\u{A}\t\u{85}\u{85}\u{A}\u{2028}\u{2029}//\u{2001}\u{2002}\u{202F}sWiFt-tOoLs-vErSiOn:\u{1680}\t\u{2000}\u{200A} \u{2002}\u{202F}9995.5.2\nkkk\n"   : (9995, 5, 2, "9995.5.2"),
+            "\n// swift-tools-version:6.1"                            : (6, 1, 0, "6.1.0"),
+            "\n\n// swift-tools-version:6.2-dev"                      : (6, 2, 0, "6.2.0"),
+            "\n\n\n// swift-tools-version:5.8.0"                      : (5, 8, 0, "5.8.0"),
+            "\n\n\n\n// swift-tools-version:6.8.0-dev.al+sha.x"       : (6, 8, 0, "6.8.0"),
+            "\n\n\n\n\n// swift-tools-version:7.1.2"                  : (7, 1, 2, "7.1.2"),
+            "\n\n\n\n\n\n// swift-tools-version:8.1.2;"               : (8, 1, 2, "8.1.2"),
+            "\n\n\n\n\n\n\n// swift-tools-vErsion:9.1.2;;;;;"         : (9, 1, 2, "9.1.2"),
+            "\n\n\n\n\n\n\n\n// swift-tools-version:6.1.2;x;x;x;x;x;" : (6, 1, 2, "6.1.2"),
+            "\n\n\n\n\n\n\n\n\n// swift-toolS-version:5.5.2;hello"    : (5, 5, 2, "5.5.2"),
+            "\n\n\n\n\n\n\n\n\n\n// sWiFt-tOoLs-vErSiOn:6.5.2\nkkk\n" : (6, 5, 2, "6.5.2"),
```

Comment:
I forgot to mention this earlier: These test cases might need a closer look.

The major version numbers were not changed by subtracting 9990 from the previous value. Because most of the resulted version would be < Swift 5.4, and those versions didn't pass tests locally.

Could someone help me check if this might be a regression?

Reply:
I tested the tools version comments with leading empty lines using Swift 5.3.2. They're not supported, so this doesn't seem like a regression. I modified the comment right above the marked lines of code to clarify it (3d383d51bc385bd2b7b4df7496e5807a2d3c4ac8). I'm keeping this conversation unresolved for a day, so that it's visible.

Reply:
Thanks, and sorry it took a while to reply — I don't have the context here, but if this isn't a regression compared with 5.3.x, then I don't think this is an issue.  Thanks for your diligence here.